helm/sealed-secrets/templates/deployment.yaml

```diff
@@ -45,6 +45,7 @@ spec:
           args:
             - --key-prefix
             - {{ .Values.secretName | quote }}
+            - --controller-port-name=http
```

Comment:
Please bump the patch version in the [Chart.yaml](https://github.com/bitnami-labs/sealed-secrets/blob/main/helm/sealed-secrets/Chart.yaml)

Reply:
Follow up question: you're implementing this flag for `kubeseal` but the `controller` ignores it. Am I right?